src/nbl/video/COpenGLCommon.h

```diff
@@ -14,250 +14,251 @@ namespace nbl
 namespace video
 {
 
+// @Crisspl these switches are woefully incomplete on all 3 functions
```

Comment:
complete them... we'll need them to the format support

Reply:
already done, forgot to remove the comment

Reply:
k